src/features/list-of-lessons/redux/reducer.js

```diff
@@ -0,0 +1,23 @@
+import types from './action-types.js';
+
+const INITIAL_STATE = {
+  lessonTheme: '',
+  lessonDate: '',
+};
+
+export const listOfLessonsReducer = (state = INITIAL_STATE, action) => {
+  switch (action.type) {
```

Comment:
why do u put the data in the store? can we store it in state ?

Reply:
changed with using useSate and useEffect